node/src/accountant/mod.rs

```diff
@@ -3261,7 +3517,12 @@ mod tests {
             last_paid_timestamp: past_payable_timestamp_2,
             pending_payable_opt: None,
         };
-        let pending_payable_scan_interval = 200; // should be slightly less than 1/5 of the time until shutting the system
+        let qualified_payables = make_guaranteed_qualified_payables(
+            vec![account_1.clone(), account_2.clone()],
+            &DEFAULT_PAYMENT_THRESHOLDS,
+            now,
+        );
+        let pending_payable_scan_interval = 200; // Should be slightly less than 1/5 of the time until shutting the system
```

Comment:
I can't add the comment at the main line, so I'll add here
<img width="838" alt="Screenshot 2024-07-12 at 3 06 42 PM" src="https://github.com/user-attachments/assets/09c1ca0f-a547-45db-a097-f335d1ccd280">